.ci/docker-compose-file/docker-compose-clickhouse.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3.9'
+
+services:
+  clickhouse:
+    container_name: clickhouse
+    image: clickhouse/clickhouse-server:latest
```

Comment:
Suggestion: pin a specific version to avoid this breaking after a new version is released.

Reply:
Fixed